docs/src/site/markdown/index.md.vm

```diff
@@ -0,0 +1,134 @@
+---
+title: Apache Software Foundation Parent POM
+author: 
+  - Benson Margulies
+  - Hervé Boutemy
+  - Karl Heinz Marbaise
+date: 2016-05-09
```

Comment:
Do we need the authors / date information?  Those are available in git history with more accuracy and much more details.

Reply:
This is orthogonal to the conversion.

Reply:
Right, but if you don't plan to remove it, shouldn't you add your name to keep it up-to-date ?

Reply:
No, as I haven't changed the content, just the format.

Reply:
Ok, as you wish.  I don't really care much.  Fwiw, I don't consider this PR change as trivial.